setup.cfg

```diff
@@ -57,7 +57,7 @@ python_version = 3.7
+exclude=contract_api/tests/test_contract_api.py|fipa/tests/test_fipa.py|default/tests/test_default.py|gym/tests/test_gym.py|http/tests/test_http.py|ledger_api/tests/test_ledger_api.py|oef_search/tests/test_oef_search.py|state_update/tests/test_state_update.py|tac/tests/test_tac.py|erc1155/tests/test_contract.py|gym/tests/test_handlers.py|gym/tests/test_rl_agent.py|gym/tests/test_task.py|gym/tests/test_helpers.py|gym/tests/test_dialogues.py|gym/tests/intermediate_class.py|erc1155_deploy/tests/test_behaviours.p|erc1155_deploy/tests/test_dialogues.py|erc1155_deploy/tests/test_handlers.py|erc1155_deploy/tests/test_handlers.py|erc1155_deploy/tests/test_strategy.py|erc1155_client/tests/test_behaviours.py|erc1155_client/tests/test_handlers.py|erc1155_client/tests/test_dialogues.py|erc1155_client/tests/test_strategy.py|generic_buyer/tests/test_handlers.py|generic_buyer/tests/test_behaviours.py|generic_buyer/tests/test_dialogues.py|generic_buyer/tests/test_models.py|erc1155_deploy/tests/intermediate_class.py|generic_seller/tests/test_handlers.py|generic_seller/tests/test_models.py|generic_seller/tests/test_dialogues.py|generic_seller/tests/test_behaviours.py|ledger/tests/test_ledger_api.py|local/tests/test_search_services.py|echo/tests/test_handlers.py|http_echo/tests/test_dialogues.py|http_client/tests/test_http_client.py|http_server/tests/test_http_server.py|echo/tests/test_dialogues.py|echo/tests/test_behaviours.py|local/tests/test_misc.py|signing/tests/test_signing.py|tendermint/tests/test_tendermint.py|stub/tests/test_stub.py|ledger/tests/conftest.py|erc1155_client/tests/intermediate_class.py|gym/tests/helpers.py|acn/tests/test_acn.py|packages/valory/connections/test_libp2p
```

Comment:
I've added `packages/valory/connections/test_libp2p` to exclude list because there are a lot of type issues but the good thing is most of them are caused by functions not being annotated with returns types properly.